packages/calcite-components/src/components/segmented-control/segmented-control.tsx

```diff
@@ -332,4 +329,15 @@ export class SegmentedControl
       match.focus();
     }
   }
+
+  private setUpItems(): void {
+    const items = this.getItems();
+    const lastChecked = items.filter((item) => item.checked).pop();
```

Comment:
could use array.find as well right?

Reply:
Good idea, but maybe once Safari 17 is out. I need to find the last checked item and Safari added support for `findLast` in v15.4. 🧭🥲